plugins/PagePerformance/tests/UI/PagePerformance_spec.js

```diff
@@ -154,5 +156,60 @@ describe("PagePerformance", function () {
         expect(await pageWrap.screenshot()).to.matchImage('pagetitle_overlay');
     });
 
+  it("should not show row evolution icon in page urls and page titles reports when in Behaviour > Performance page", async function () {
+    await page.goto("?" + urlBase + "#?" + generalParams + "&category=General_Actions&subcategory=PagePerformance_Performance");
 
+    // Check page report
+    let row = await page.waitForSelector(pageUrlsReportId + ' .dataTable tbody tr:first-child');
+    await row.hover();
+    await page.waitForTimeout(50);
+    pageWrap = await page.$(pageUrlsReportId);
+
+    let rowActions = await row.$('.dataTableRowActions');
+    expect(rowActions).to.not.equal(null);
+
+    let rowActionLinks = await row.$$('.dataTableRowActions a');
+    expect(rowActionLinks.length).to.equal(4);
+
+    let icon = await pageWrap.$('.actionRowEvolution');
+    expect(icon).to.equal(null);
+
+    // Check Page Titles report
+    row = await page.waitForSelector(pageTitleReportId + ' .dataTable tbody tr:first-child');
+    await row.hover();
+    await page.waitForTimeout(50);
+    pageWrap = await page.$(pageTitleReportId);
+
+    rowActions = await row.$('.dataTableRowActions');
+    expect(rowActions).to.not.equal(null);
+
+    rowActionLinks = await row.$$('.dataTableRowActions a');
+    expect(rowActionLinks.length).to.equal(3);
+
+    icon = await pageWrap.$('.actionRowEvolution');
```

Comment:
Looking for the icon within the `row` element is a bit more precise.

```suggestion
    icon = await row.$('.actionRowEvolution');
```